packages/compile-solidity/src/run.js

```diff
@@ -12,8 +12,11 @@ async function run(rawSources, options, internalOptions = {}) {
     return null;
   }
 
-  const language = internalOptions.language || "Solidity"; //could also be "Yul"
-  const noTransform = internalOptions.noTransform; //turns off project root transform
+  const {
```

Comment:
Oh, I forgot you can do this.  Thanks for cleaning this up!

Reply:
you bet!